angular/projects/spark-angular/src/lib/components/sprk-modal/sprk-modal.component.ts

```diff
@@ -40,43 +42,53 @@ import * as _ from 'lodash';
             [attr.data-analytics]="closeAnalyticsString"
             (click)="closeModal($event)"
           >
-            <sprk-icon
-              iconType="close"
-            ></sprk-icon>
+            <sprk-icon iconName="close"></sprk-icon>
           </button>
         </header>
 
         <div
```

Comment:
Didn't update this to use `<sprk-stack>` because it messes with the css selectors and didn't want to dive into updating that as well. Can be re-looked at as part of the Modal API upgrades.